theano/tensor/nnet/tests/test_conv3d2d.py

```diff
@@ -190,3 +217,87 @@ def test_conv3d(border_mode, mode=mode_without_gpu, shared=theano.tensor._shared
     filters = numpy.random.rand(Nf, Tf, C, Hf, Wf).astype('float32')
     utt.verify_grad(lambda s, f: conv3d(s, f, border_mode=border_mode),
                     [signals, filters], eps=1e-1, mode=mode)
+
+
+def get_conv3d_test_cases(experiment='fwd'):
```

Comment:
I don't remember exactly why I added them, except that I was probably afraid that there might be a few edge cases that are not included in the original tests, but I can't see what they are. I'll remove the new tests.